pygmt/modules.py

```diff
@@ -108,7 +122,16 @@ def info(table, **kwargs):
                     [fname, build_arg_string(kwargs), "->" + tmpfile.name]
                 )
                 lib.call_module("info", arg_str)
-            return tmpfile.read()
+            result = tmpfile.read()
+
+        if any(arg in kwargs for arg in ["C", "I", "T"]):
+            # Converts certain output types into a numpy array
+            # instead of a raw string that is less useful.
+            result = np.loadtxt(
+                re.sub(pattern="-R|-T|/", repl=" ", string=result).splitlines()
+            )
```

Comment:
Is it OK to check if a string starts with `-R` or `-T`?

Reply:
Yes, with `str.startswith` (see https://stackoverflow.com/questions/20461847/str-startswith-with-a-list-of-strings-to-test-for#20462341). I try to avoid regex as much as I can, but after several ways of experimenting on how to handle 1) `-Rw/e/s/n`; 2) `-Tmin/max/inc`; 3) `xmin xmax ymin ymax` 4) `x1 y1 x2 y2` (in several rows), this seems to be the most succinct way. I can try again though.

Reply:
Actually, this works too :grin::

https://github.com/GenericMappingTools/pygmt/blob/a52127fae0400701b60119704f5fc787f2fb55a2/pygmt/modules.py#L128-L130

Reply:
Great!